docs/en-us/open-api.md

```diff
@@ -11,6 +11,9 @@ description: Open API Guide
   - [Listen for configurations](#1.2)
   - [Publish configuration](#1.3)
   - [Delete configuration](#1.4)
+  - [Query the list history configuration](#1.5)
+  - [Query the detailed configuration history infomation](#1.6)
+  - [Query previous config history information](#1.7)
```

Comment:
Maybe some trouble for English description.

For example: `Query history configuration list` or `Query list of history configuration`.

I think the three title need be translate again.